packages/usa-banner/src/styles/_usa-banner.scss

```diff
@@ -143,6 +112,7 @@ $banner-icon-close: (
 }
 
 .usa-banner__header-action {
+  @include u-margin-y(0);
```

Comment:
This narrows the space between paragraphs by setting margin-top:0. Prior version had margin-top: 2px.

Reply:
Yes, it's added to the action on mobile **only**. It seemed small enough to remove, but I've added back in [86aa75ca941cd168ce3ffeb73555fed4428773e8].

![image](https://user-images.githubusercontent.com/3385219/189144870-01559dea-3da9-47d6-a1aa-4a3a301406a3.png)